plugins/jcache/src/main/java/org/radargun/service/JCacheOperations.java

```diff
@@ -0,0 +1,90 @@
+package org.radargun.service;
+
+import org.radargun.traits.BasicOperations;
+import org.radargun.traits.ConditionalOperations;
+
+/**
+ * @author Matej Cimbora
+ */
+public class JCacheOperations implements BasicOperations, ConditionalOperations {
+
+   private JCacheService service;
+
+   public JCacheOperations(JCacheService service) {
+      this.service = service;
+   }
+
+   @Override
+   public <K, V> Cache<K, V> getCache(String cacheName) {
```

Comment:
So, you're silently ignoring the requested cacheName here? That's not correct. JCache CacheManager seems to support multiple caches.